src/iterative_sorting/iterative_sorting.py

```diff
@@ -20,6 +27,12 @@ def selection_sort( arr ):
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
+    for _ in range(0,100):
```

Comment:
the traditional way of doing this sort is to initialize a variable, for example "sorted" and set it to True. and then use that variable to use a while loop. 

sorted = True
while sorted:
    sorted = False
    (your for loop goes here)
    (after switching you set sorted to True, else the while loop breaks because everything has been sorted)

your method works but it adds a bunch of steps that aren't necessary. using this sorted variable as a flag to know when the list has been sorted is going to be very important to solving the sprint challenge this week (hint hint wink wink)!

Reply:
Thank you Austin! It was really helpful. I always appreciate your input and feel great that you check the code everyday. :-)